dss-api.yml

```diff
@@ -56,13 +56,20 @@ basePath: /v1
 produces:
   - application/json
 securityDefinitions:
-  googAuth:
+  fusilladeAuth:
```

Comment:
I think we should change `fusilladeAuth` to `dcpAuth`. "fusillade" is an implementation detail of sorts :)

Reply:
done